linera-service/src/client.rs

```diff
@@ -922,7 +922,49 @@ impl NodeService {
                 requiredApplicationIds: {json_required_applications_ids}) \
             }}"
         );
-        let data = self.query_node(&query).await;
+        self.query_node(&query).await
+    }
+
+    pub async fn create_application_tuple(
+        &self,
+        chain_id: &ChainId,
+        bytecode_id: String,
+        parameters: String,
+        argument: String,
+        required_application_ids: Vec<String>,
+    ) -> (String, ApplicationId) {
+        let data = self
+            .create_application_impl(
+                chain_id,
+                bytecode_id,
+                parameters,
+                argument,
+                required_application_ids,
+            )
+            .await;
+        (
+            serde_json::from_value(data["createApplication"].clone()).unwrap(),
+            serde_json::from_value::<ApplicationId>(data["createApplication"].clone()).unwrap(),
```

Comment:
It looks like there's only one call site for this; should we just parse the application ID there, instead of having two versions of this method?

Reply:
nit: Remove the type `::<ApplicationId>`. In general, we don't add type annotations unless it's required for compilation.

Reply:
@afck True, I can do that
@ma2bd So if I remove the `::<ApplicationId>`, it will compile, but it'll be the same call as the one above it, which defaults to a `String` return type, but I want `ApplicationId`. I don't think this will be needed at all though once I do what @afck mentioned, so we should be ok here